src/client/e2e/limit-checker.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@playwright/test"
+import { Page } from "playwright"
+
+import { test } from "./fixtures"
+import { OPENFIN_PROJECT_NAME } from "./utils"
+
+test.describe("Limit Checker", () => {
+  let tilePage: Page
+  let blotterPage: Page
+  let limitCheckerPage: Page
+
+  test.beforeAll(async ({ fxPagesRec, limitCheckerPageRec }, testInfo) => {
+    if (testInfo.project.name === OPENFIN_PROJECT_NAME) {
+      tilePage = fxPagesRec["fx-tiles"]
+      blotterPage = fxPagesRec["fx-blotter"]
+      limitCheckerPage = limitCheckerPageRec
+    }
+  })
+
+  // eslint-disable-next-line no-empty-pattern
+  test("Trade notional is capped by limit checker", async ({}, testInfo) => {
+    test.skip(testInfo.project.name !== OPENFIN_PROJECT_NAME, "Openfin Only")
+
+    const limitTableFirstRowCells = limitCheckerPage
+      .locator(`[role="grid"] > div`)
+      .nth(1)
+      .locator("div")
+
+    const tradeBlotterFirstRowCells = blotterPage
+      .locator("[role=grid] > div")
+      .nth(1)
+      .locator("div")
+
+    const limitCheckInput = limitCheckerPage.getByLabel("EUR/USD")
+
+    const limitId = await limitTableFirstRowCells.nth(1).innerText()
```

Comment:
It would be great if we try to avoid using `nth` as much as possible etc.. and use getByText or by label etc.. to make more user readable

Reply:
nth is quite useful for the grid, no labels and because getByText is a contains check and not an exact check searching for things like id's tends to return a lot of results and causes problems. But I have extracted the assertions into more readable helper functions which we might be able to reuse in other grid tests.